src/sanitizer.js

```diff
@@ -198,11 +198,16 @@ export function sanitizeHtml(html) {
         } else {
           attribs = scrubbed.attribs;
           // Restore some of the attributes that AMP is directly responsible
-          // for, such as "on".
+          // for, such as "on" or amp-bind's bracketed attributes.
           for (let i = 0; i < attribs.length; i += 2) {
-            if (WHITELISTED_ATTRS.indexOf(attribs[i]) != -1) {
+            const attrib = attribs[i];
+            if (WHITELISTED_ATTRS.indexOf(attrib) != -1) {
               attribs[i + 1] = savedAttribs[i + 1];
-            } else if (attribs[i].search(WHITELISTED_ATTR_PREFIX_REGEX) == 0) {
+            } else if (attrib.search(WHITELISTED_ATTR_PREFIX_REGEX) == 0) {
+              attribs[i + 1] = savedAttribs[i + 1];
+            } else if (attrib[0] == '[' && attrib[attrib.length - 1] == ']') {
+              // Completely restore bind attribute values, as sanitizer does not recognize
```

Comment:
I've added `text` and `class` to the whitelist. I spent about an hour digging into ways to change the sanitizer to support bind attributes. Rather than making multiple changes to check the contents of bracketed attribute names, I think the way to go here is to strip off the brackets at the beginning, treat the attributes exactly like their non-bind counterparts (treat [onclick] as onclick etc.), and then add the brackets back on at the end. 

I've also added corresponding unit tests. Let me know if you'd like me to add more.